src/pages/About/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
Did you explicitly not link the about page to the Header component? 
`<Link to='/' className={classes.links}>About</Link>`

Reply:
Thank you. I've now added it to the Header and Footer.